README.md

```diff
@@ -40,7 +45,9 @@ In an AJAX POST request, what does the argument of the `.done()` callback - in t
   });
 ```
 ### Question 6
-Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`. 
+Suppose you had the following form in your HTML file. Use jQuery to write a single line of code that takes whatever the user entered in the textbox and saves it to the variable `user_input`.
+
+var $user_input = $('input[type="text"]');
```

Comment:
Not sure that would work as a selector, and in any case, it would not get you what the user typed in. For that you need the .val() method. So `var $user_input = $('#color').val()`.